sematic/ui/packages/main/src/components/RunTime.tsx

```diff
@@ -1,10 +1,21 @@
 import { Typography } from "@mui/material";
 import { Run } from "@sematic/common/src/Models";
 
+function InstantRunTime() {
+  return (
+    <Typography fontSize="small" color="GrayText">
+      {"<1s"}
+    </Typography>
+  );
+}
+
 export function RunTime(props: { run: Run; prefix?: string }) {
   const { run, prefix = "" } = props;
   let startedAt = new Date(run.started_at || run.created_at);
   let endedAt = new Date();
+  if (run.original_run_id !== null) {
+    return <InstantRunTime />;
```

Comment:
How about displaying a message saying the duration is unknown whenever one of the times is not present?

Reply:
No, in this case `original_run_id !== null`, we know the run executed instantly, so we should not make it look like something is unknown.

As a fallback in a general case, possibly.

Reply:
Added a fallback case in case `run.started_at` and `run.created_at` are both `null`.

Reply:
Yes, commented on the wrong line. Thank you!